_posts/v2/2014-11-12-ita-faqs.markdown

```diff
@@ -7,6 +7,8 @@ published: true
 
 #FAQs on Exporting API
 
+######Data set updated monthly
```

Comment:
We don't import these periodically at present, so it's technically not correct to say this.